sippy-ng/src/component_readiness/ComponentReadiness.js

```diff
@@ -502,6 +504,15 @@ export default function ComponentReadiness(props) {
     setRedOnlyChecked(false)
   }
 
+  const regressedTests = mergeRegressedTests(data)
+  const [regressedTestDialog = false, setRegressedTestDialog] = useQueryParam(
+    'regressedModal',
+    BooleanParam
+  )
+  const closeRegressedTestsDialog = () => {
+    setRegressedTestDialog(false)
```

Comment:
```suggestion
    setRegressedTestDialog(false, 'replace')
```

This should fix the history problems you were having

Reply:
Turned out it is 'replaceIn'. Using 'replace', I was seeing all kinds of weird behaviors. I think it replaced all parameters.